SpatialGDK/Source/SpatialGDK/Public/EngineClasses/SpatialLoadBalanceEnforcer.h

```diff
@@ -3,32 +3,36 @@
 #pragma once
 
 #include <WorkerSDK/improbable/c_worker.h>
+
 #include "CoreMinimal.h"
 
 DECLARE_LOG_CATEGORY_EXTERN(LogSpatialLoadBalanceEnforcer, Log, All)
 
 class SpatialVirtualWorkerTranslator;
-class USpatialSender;
 class USpatialStaticComponentView;
 
 class SpatialLoadBalanceEnforcer
 {
 public:
-	SpatialLoadBalanceEnforcer();
+	struct AclWriteAuthorityRequest
+	{
+		Worker_EntityId EntityId = 0;
+		FString OwningWorkerId;
+	};
 
-	void Init(const FString &InWorkerId, USpatialStaticComponentView* InStaticComponentView, USpatialSender* InSpatialSender, SpatialVirtualWorkerTranslator* InVirtualWorkerTranslator);
-	void Tick();
+	SpatialLoadBalanceEnforcer(const FString &InWorkerId, USpatialStaticComponentView* InStaticComponentView, SpatialVirtualWorkerTranslator* InVirtualWorkerTranslator);	
```

Comment:
You might have to also change GetPhysicalWorkerForVirtualWorker to be const as well

Reply:
Made `GetPhysicalWorkerForVirtualWorker` const